tests/flakyfunctional/events/01-task/flow.cylc

```diff
@@ -25,7 +25,6 @@
 [runtime]
     [[root]]
         # make the event handler log available to gcylc:
-        extra log files = {{ EVNTLOG }}
```

Comment:
I think the comment on the line above should go as well?

Reply:
Yep. Well Spotted. 🐆